lib/commands/list-subscriptions.js

```diff
@@ -21,28 +21,40 @@ module.exports = async (req, res, next) => {
     where: { channelId: command.channel_id },
   });
 
-  let repositories = await Promise.all(subscriptions.map(async (subscription) => {
+  let resources = await Promise.all(subscriptions.map(async (subscription) => {
     const github = await robot.auth(subscription.Installation.githubId);
+    let githubResource;
+
     try {
-      const repository = await github.repos.getById({ id: subscription.githubId });
-      return repository.data;
+      githubResource = await github.repos.getById({ id: subscription.githubId });
+      return githubResource.data;
     } catch (err) {
-      req.log.error({ err, repoId: subscription.githubId }, 'Could not find repository for subscription');
       if (err.code !== 404) {
         throw err;
       }
     }
+
+    try {
+      githubResource = await github.users.getById({ id: subscription.githubId });
+      return githubResource.data;
+    } catch (err) {
+      if (err.code !== 404) {
+        throw err;
+      }
+    }
+
+    req.log.error({ githubId: subscription.githubId }, 'Could not find resource for subscription');
   }));
 
   // remove undefined
-  repositories = repositories.filter(repo => repo);
+  resources = resources.filter(repo => repo);
 
   if (command.text === 'list') {
-    return command.respond(new SubscriptionList(repositories, command.channel_id));
+    return command.respond(new SubscriptionList(resources, command.channel_id));
   }
 
   const response = (new Help(command.command, command.subcommand)).toJSON();
-  const list = new SubscriptionList(repositories, command.channel_id);
+  const list = new SubscriptionList(resources, command.channel_id);
   response.attachments.push(list.toJSON().attachments[0]);
   return command.respond(response);
 };
```

Comment:
I think I'd also prefer if for the `/github subscribe list` command we could list the number of repositories and the number of "accounts" as separate counts and lists as opposed to adding them together.

For what it's worth, I'm also not completely in love with the "resources" name here. Is there another name that we could use here?